src/rpcmining.cpp

```diff
@@ -585,7 +593,17 @@ Value getblocktemplate(const Array& params, bool fHelp)
         entry.push_back(Pair("fee", pblocktemplate->vTxFees[index_in_template]));
         entry.push_back(Pair("sigops", pblocktemplate->vTxSigOps[index_in_template]));
```

Comment:
According to https://github.com/bitcoin/bips/blob/master/bip-0022.mediawiki#transactions-object-format , absent ``"fee"`` or ``"sigops"`` keys mean "unknown".